homeassistant/components/vallox/config_flow.py

```diff
@@ -82,9 +83,15 @@ async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Handle the initial step."""
+        integration = await async_get_integration(self.hass, DOMAIN)
+
         if user_input is None or user_input[CONF_HOST] is None:
```

Comment:
When can the host be None?

Reply:
Good question, I don't think it can. Removed that part of the check. 👍